presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -352,6 +353,14 @@ public void checkCanShowSchemas(SystemSecurityContext context, String catalogNam
         return schemaNames;
     }
 
+    @Override
+    public void checkCanShowCreateTable(SystemSecurityContext context, CatalogSchemaTableName table)
+    {
+        if (!canAccessCatalog(context.getIdentity(), table.getCatalogName(), ALL)) {
```

Comment:
(no change requested) it is strange that one has `ALL` and other has `OWNERSHIP`. Should I file an issue for that?

Reply:
That is weird.  I actually like to extend the system level file rules to support everything in the connector level.